aws/resource_aws_launch_template.go

```diff
@@ -559,27 +570,27 @@ func getBlockDeviceMappings(m []*ec2.LaunchTemplateBlockDeviceMapping) []interfa
 	s := []interface{}{}
 	for _, v := range m {
 		mapping := map[string]interface{}{
-			"device_name":  *v.DeviceName,
-			"virtual_name": *v.VirtualName,
+			"device_name":  aws.StringValue(v.DeviceName),
+			"virtual_name": aws.StringValue(v.VirtualName),
 		}
 		if v.NoDevice != nil {
 			mapping["no_device"] = *v.NoDevice
 		}
 		if v.Ebs != nil {
 			ebs := map[string]interface{}{
-				"delete_on_termination": *v.Ebs.DeleteOnTermination,
-				"encrypted":             *v.Ebs.Encrypted,
-				"volume_size":           *v.Ebs.VolumeSize,
-				"volume_type":           *v.Ebs.VolumeType,
+				"delete_on_termination": aws.BoolValue(v.Ebs.DeleteOnTermination),
+				"encrypted":             aws.BoolValue(v.Ebs.Encrypted),
+				"volume_size":           aws.Int64Value(v.Ebs.VolumeSize),
```

Comment:
Any of these `aws.Int64Value()` calls will need to be wrapped in `int(aws.Int64Value())` since Terraform's `TypeInt` is a regular `int`